Person.sql

```diff
@@ -0,0 +1,20 @@
+create table Person
+(
+	id INT UNIQUE,
+	name VARCHAR(50),
+	surname VARCHAR(50),
+	age INT NOT NULL,
+	passport_number VARCHAR(50) UNIQUE
+	
+	INSERT INTO Person (id, name, surname, age, passport_number) VALUES (1, 'Ivan', 'Ivanov', 57, 'MC56789043');
```

Comment:
Why lines 9-19 are in the table creation brackets?

Reply:
first brackets to determine the order in which data is entered into the database. Second -  just syntax when populating the table with data. 
I know, i can create the table with out first brackets, but It was simply easier for me to write down the order than to remember it myself 
or you mean something else?

Reply:
Yes.
I mean the INSERT script is placed inside `create table` script - it's incorrect